source/features/show-forks-in-search.tsx

```diff
@@ -0,0 +1,33 @@
+import './show-forks-in-search.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import SearchQuery from '../github-helpers/search-query';
+
+function init(): void {
+	const includeForks = new SearchQuery(location.search).includes('fork:true');
+	const checkbox = <label className="float-right"><input className="rgh-show-forks-in-search-checkbox mr-1" type="checkbox" checked={includeForks}/>Include forks</label>;
```

Comment:
```suggestion
	const checkbox = (
		<label className="float-right">
			  <input className="rgh-show-forks-in-search-checkbox mr-1" type="checkbox" checked={includeForks}/>
			  Include forks
		</label>
	);
```

Reply:
I remember linter would yell at me for this before, but it didn't so I passed it for now 😄

Reply:
If you find incomprehensible/difficult build/lint errors you can ask for help, as long as the logic works.